from quickdataanalysis.exceptions.quickdataanalysis_exception import QuickdataAnalysis